### ISSUE TITLE
Changes to be done to menu and page to be responsive

### ISSUE THREAD

**Issue by principlebrothers** [norbiboy1/Portfolio-setup-and-mobile-version-skeleton#11]
**Excellent work done**

##Few Changes require ♻️ 

- It will be appropriate if you use an Html entity instead of the hamburger-icon
- It will be more appropriate if the page is responsive to help with Search Engine Optimization (SEO)

Thank you 😁

**Comment by r-ahmed2022** [comment]
Yes, the Html entity has advantage that it could be customised better.
